Makefile

```diff
@@ -24,6 +24,9 @@ dependencies:
 	# We use 'go get' here because 'go install' will not build out of our vendored mock repo.
 	go get github.com/golang/mock/mockgen
 
+	# Necessary to generate thrift files, in addition to manually installing thrift via homebrew
+	go get github.com/apache/thrift/lib/go/thrift
```

Comment:
Shouldn't. We can have copies in both GOPATH and vendor. GOPATH is used with the thrift compiler, and vendor is used for the thrift lib in the code, I think it's fine.